pkg/app/app.go

```diff
@@ -15,18 +20,32 @@ const (
 )
 
 type App struct {
-	Window  *glfw.Window
-	Context *g.Context
-	FpsCounter *utils.FPSCounter
+	Window         *glfw.Window
+	Context        *g.Context
+	FpsCounter     *utils.FPSCounter
+	FpsCounterText *ui.Text
 }
 
-func InitApp(windowWidth int, windowHeight int, windowCentered bool, windowTitle string) (*App) {
+func InitApp(windowWidth int, windowHeight int, windowCentered bool, windowTitle string) *App {
 	runtime.LockOSThread()
 	app := &App{}
 	app.Window = initWindow(windowWidth, windowHeight, windowTitle)
 	app.Context = &g.Context{}
 	app.Context.SetOrtho2DProjection(windowWidth, windowHeight, 1, windowCentered)
 	app.FpsCounter = &utils.FPSCounter{}
+
+	font := ui.NewFontFromFiles(
```

Comment:
Showing the FPS count looks like the job of an application using the library. But since it can help with the development I'm fine having it here

Reply:
yeah, we can have it behind a bool argument